olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <boost/optional.hpp>
```

Comment:
boost is not a system include.

Reply:
We definitely need to re-conigure our clang-format file to not manually change the order of includes